### PR TITLE
Update RegularAcrticle to Typescript

### PR DESCRIPTION
**Pull request by ssabbar**

### CODE REVIEW

File: src/interface/news/RegularArticle.tsx [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
 
 import Contributor from 'interface/ContributorButton';
 import makeNewsUrl from 'interface/news/makeUrl';
 import Panel from 'interface/others/Panel';
+import { ContributorType } from 'interface/ContributorButton/ContributorButton';
 
-const RegularArticle = props => {
+interface Props {
+  title: ReactNode;
+  bodyStyle?: object;
```
Comment (guyius) [inline src/interface/news/RegularArticle.tsx]: This should be `React.CSSProperties`

Comment (ssabbar) [reply src/interface/news/RegularArticle.tsx]: I used object because the Panel component used object as a type for bodystyle prop

Comment (emallson) [reply src/interface/news/RegularArticle.tsx]: That is something that can be cleaned up as part of this PR. CSS props should have the correct type.

Comment (ssabbar) [reply src/interface/news/RegularArticle.tsx]: I've made the changes

**Comment by guyius** [comment]
Can't approve, but LGTM!

**Comment by ssabbar** [comment]
Thanks :)

**Pull request merged**